frontend/next.config.js

```diff
@@ -87,4 +90,27 @@ module.exports = {
       },
     ];
   },
+  sassOptions: {
+    functions: {
+      /**
+       * Copied from node_modules\@blueprintjs\core\scripts\sass-custom-functions.js
+       *
+       * Sass function to inline a UI icon svg and change its path color.
+       *
+       * Usage:
+       * svg-icon("16px/icon-name.svg", (path: (fill: $color)) )
+       *
+       * @see https://github.com/palantir/blueprint/issues/4759#issuecomment-1112218581
+       */
+      "svg-icon($path, $selectors: null)": inliner(
+        path.join(__dirname, "blueprint-icons/icons"), // Replaced path to icons
+        {
+          // minimal "uri" encoding is smaller than base64
+          encodingFormat: "uri",
+          // run through SVGO first
+          optimize: true,
+        }
+      ),
+    },
+  },
```

Comment:
I removed `swcMinify: true`, because it breaks the build

See: https://github.com/vercel/next.js/issues/38436#issuecomment-1220155395